aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1494,7 +1520,7 @@ impl AptosVM {
             )));
 
         let resolver = vm.as_move_resolver(&state_view);
-        let mut session = vm.new_session(&resolver, SessionId::Void);
+        let mut session = vm.0.new_session(&resolver, SessionId::Void);
```

Comment:
let's name .0 with something?

Reply:
I think we should get rid of impl completely😅 will change it to vm_impl in the next PR